src/llama_stack_client/lib/cli/inference/inference.py

```diff
@@ -19,30 +19,63 @@ def inference():
 
 
 @click.command("chat-completion")
-@click.option("--message", required=True, help="Message")
+@click.option("--message", help="Message")
 @click.option("--stream", is_flag=True, help="Streaming", default=False)
+@click.option("--session", is_flag=True, help="Start a Chat Session", default=False)
 @click.option("--model-id", required=False, help="Model ID")
 @click.pass_context
 @handle_client_errors("inference chat-completion")
-def chat_completion(ctx, message: str, stream: bool, model_id: Optional[str]):
+def chat_completion(ctx, message: str, stream: bool, session: bool, model_id: Optional[str]):
     """Show available inference chat completion endpoints on distribution endpoint"""
+    if not message and not session:
+        click.secho(
+            "--message is required if not starting a chat session",
+            fg="red",
+        )
+        raise click.exceptions.Exit(1)
     client = ctx.obj["client"]
     console = Console()
 
     if not model_id:
         available_models = [model.identifier for model in client.models.list() if model.model_type == "llm"]
         model_id = available_models[0]
 
-    response = client.inference.chat_completion(
-        model_id=model_id,
-        messages=[{"role": "user", "content": message}],
-        stream=stream,
-    )
-    if not stream:
-        console.print(response)
-    else:
-        for event in EventLogger().log(response):
-            event.print()
+    messages = []
+    if message:
+        messages.append({"role": "user", "content": message})
+        response = client.inference.chat_completion(
+            model_id=model_id,
+            messages=messages,
+            stream=stream,
+        )
+        if not stream:
+            console.print(response)
+        else:
+            for event in EventLogger().log(response):
+                event.print()
+    if session:
+        chat_session(client=client, model_id=model_id, messages=messages, console=console)
+
+
+def chat_session(client, model_id: Optional[str], messages: List[Dict[str, str]], console: Console):
+    """Run an interactive chat session with the served model"""
+    while True:
+        try:
+            message = input(">>> ")
```

Comment:
you could also stop the session if the user types in `quit`

Reply:
adding  "quit" and "\q"